modules/Layout/Header/SearchWidget/components/MainPanel.tsx

```diff
@@ -9,6 +10,12 @@ import styles from './MainPanel.module.scss';
 
 function MainPanel({ searchState, searchResults }: StateResultsProvided<AlgoliaStory>) {
     const isQuerySet = Boolean(searchState.query?.length);
+    const categories = useCategories();
+
+    // return null (render nothing if newsroom does not have any categories and there are no search results)
```

Comment:
This comment is not really necessary, it's already pretty clear from the code :)

Reply:
Sure! :)